MUXSDKStats/MUXSDKStatsTests/MUXSDKStatsTests.m

```diff
@@ -337,29 +337,27 @@ - (void) testOrientationAndRenditionChangeEventSequence {
                                     MUXSDKPlaybackEventRenditionChangeEventType,
                                     MUXSDKDataEventType,
                                     MUXSDKPlaybackEventRenditionChangeEventType,
-                                    MUXSDKDataEventType,
                                     MUXSDKPlaybackEventOrientationChangeEventType,
                                     MUXSDKDataEventType,
                                     MUXSDKPlaybackEventRenditionChangeEventType
     ];
     [self assertPlayer:playName dispatchedEventTypes:expectedEventTypes];
-    
+
     [self assertPlayer:playName dispatchedDataEvents:@{
         @(1): [NSNull null],
         @(4): @{BANDWIDTH: @(258157)},
         @(6): @{BANDWIDTH: @(1927853)},
-        @(8): @{BANDWIDTH: @(1927853),},
-        @(10): @{BANDWIDTH: @(258157)},
+        @(9): @{BANDWIDTH: @(258157)},
     }];
-    
+
     [self assertPlayer:playName dispatchedPlaybackEvents:@{
         @(0): [NSNull null],
         @(2): [NSNull null],
         @(3): @{X: @(0.0), Y: @(0.0), Z: @(90.0)},
         @(5): [NSNull null],
         @(7): [NSNull null],
-        @(9): @{X: @(0.0), Y: @(0.0), Z: @(0.0)},
-        @(11): [NSNull null],
+        @(8): @{X: @(0.0), Y: @(0.0), Z: @(0.0)},
+        @(10): [NSNull null],
```

Comment:
What changed in this `renditionchange` event?

Reply:
We send one fewer DataEvent so the indices changed:
```
    // Assert sequence of playback & data events is correct

    NSArray *expectedEventTypes = @[MUXSDKPlaybackEventViewInitEventType,
                                    MUXSDKDataEventType,
                                    MUXSDKPlaybackEventPlayerReadyEventType,
                                    MUXSDKPlaybackEventOrientationChangeEventType,
                                    MUXSDKDataEventType,
                                    MUXSDKPlaybackEventRenditionChangeEventType,
                                    MUXSDKDataEventType,
                                    MUXSDKPlaybackEventRenditionChangeEventType,
                                    MUXSDKPlaybackEventOrientationChangeEventType,
                                    MUXSDKDataEventType,
                                    MUXSDKPlaybackEventRenditionChangeEventType
```